defaults/main/001-kafka.yml

```diff
@@ -1,8 +1,9 @@
 ---
 # The Apache Kafka version to be downloaded and installed
 # kafka_download_base_url should be set to https://archive.apache.org/dist/kafka/ for older versions than the current
-kafka_download_base_url: http://www-eu.apache.org/dist/kafka
-kafka_version: 3.1.0
+kafka_download_base_url: https://dlcdn.apache.org/kafka
```

Comment:
A new site is now provided from the Apache Kafka documentation for downloading releases from.